packages/core/useScriptTag/index.ts

```diff
@@ -0,0 +1,126 @@
+import { isString, MaybeRef, tryOnMounted, tryOnUnmounted } from '@vueuse/shared'
+import { ref } from 'vue-demi'
+import { ConfigurableDocument, defaultDocument } from '../_configurable'
+
+export interface UseScriptTagOptions {
+  loadOnMounted?: boolean
```

Comment:
```suggestion
  immediate?: boolean
```

Prefer this naming to align with Vue's options and other functions